modules/ui/bin/jenkins-build-deploy.sh

```diff
@@ -109,11 +111,12 @@ elif [[ "${version}" == *SNAPSHOT ]]; then
   artifact_repository_id=${nexus_snapshot_repository_id}
 fi
 
-group=`fromPom main ${profile} project.groupId`
+group=`fromPom project.groupId`
 artifact=ui
-path=${nexus_repositories}/${artifact_repository_id}/`echo ${group} | sed "s/\./\//g"`/${artifact}/${version}
-zip=${project}-${artifact}-${profile}-${version}.zip
-zip_path=${path}/${zip}
+group_modified=`echo ${group} | sed "s/\./\//g"`
```

Comment:
If we still want the variable, can we name it a little more intuitively instead of `group_modified`. I don't understand this `sed` command properly, so cannot comment on the proper name.